index.js

```diff
@@ -14,5 +13,4 @@ function reveal() {
     }
   }
   
-  window.addEventListener("scroll", reveal);
-  
\ No newline at end of file
+  window.addEventListener("scroll", reveal);
```

Comment:
zrob sobie cwiczenie. 
majac tego eventListenera na scroll podepnij funkcje ktora jedyne co bedzie robic to console log. 
I zobacz jak sie zachowuje eventListener na scrollu. Odkryjesz wtedy cos ciekawego co mysle ze pomoze Ci przerobic Twoj kod